crypto/trace.c

```diff
@@ -227,8 +235,9 @@ static int set_trace_data(int category, int type, BIO **channel,
     char *curr_prefix = NULL;
     char *curr_suffix = NULL;
 
-    /* Ensure ossl_trace_init() is called */
-    OPENSSL_init_crypto(0, NULL);
+    /* Ensure do_ossl_trace_init() is called once */
+    if (!RUN_ONCE(&trace_inited, ossl_trace_init))
+      return 0;
```

Comment:
This is a nit: the indentation should have 2 more spaces before `return 0`...

Reply:
Oops; fxed.